flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,256 @@
+                                                  url=episode.get('url'),
+                                                  series_name=generate_title(request, config.get('strip_year')),
+                                                  series_season=season["seasonNumber"],
+                                                  series_episode=episode["episodeNumber"],
+                                                  series_id=tempSeries_id,
+                                                  tvdb_id=request.get('tvDbId'),
+                                                  imdb_id=request.get('imdbId'),
+                                                  ombi_request_id=request.get('id'),
+                                                  ombi_childrequest_id=childRequest["id"],
+                                                  ombi_season_id=season["id"],
+                                                  ombi_episode_id=episode["id"],
+                                                  ombi_approved=episode.get('approved'),
+                                                  ombi_available=episode.get('available'),
+                                                  ombi_requested=episode.get('requested'))
+                                    if entry.isvalid():
+                                        log.debug('Valid entry %s', entry)
+                                        if config.get('only_approved') and not entry.get('ombi_approved'):
+                                            log.verbose('Request not approved skipping: %s', entry.get('title'))
+                                        else:
+                                            if not config.get('include_available') and entry.get('ombi_available'):
+                                                log.verbose('Request already available skipping: %s', entry.get('title'))
+                                            else:
+                                                entries.append(entry)
+                                    else:
+                                        log.error('Invalid entry created? %s', entry)
+                                        continue
+            return entries
+ 
+class OmbiSet(MutableSet):
+    supported_ids = ['imdb_id', 'tmdb_id', 'ombi_id']
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string'},
+            'port': {'type': 'number', 'default': 3579},
+            'api_key': {'type': 'string'},
+            'type': {'type': 'string', 'enum': ['shows', 'seasons', 'episodes', 'movies']},            
+            'only_approved': {'type': 'boolean', 'default': True},
+            'include_available': {'type': 'boolean', 'default': False},
+            'strip_year': {'type': 'boolean', 'default': False}
+        },
+        'required': ['api_key', 'base_url', 'type'],
+        'additionalProperties': False
+    }
+
+    @property
+    def requests(self):
```

Comment:
requests isn't the best name for this, and while it technically doesn't overshadow the `request` lib used in the outer scope, it's still not a great idea to reuse the name. Maybe call it entities?

Reply:
This one is likely the result of replacing the word movies with requests. As requests are what I am pulling from Ombi not movies like couchpotato.

I could see how the name could cause problems however.